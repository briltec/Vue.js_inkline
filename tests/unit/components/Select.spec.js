// import { mount } from '@vue/test-utils';
// import Select from 'inkline/components/Select';

describe('Components', () => {
    describe('Select', () => {
        // let wrapper;

        beforeEach(() => {
            // wrapper = mount(Select);
        });

        it('should render correctly', () => {
            expect(true);
        });
    });
});
