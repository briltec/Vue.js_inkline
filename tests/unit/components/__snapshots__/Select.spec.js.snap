// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Select should render correctly 1`] = `
<div size="" tabindex="-1" class="dropdown select">
  <div class="form-input-group" aria-haspopup="list" aria-controls="dropdown-menu-f26zq37i2">
    <!---->
    <div class="form-input">
      <!---->
      <!----> <input tabindex="1" readonly="readonly" aria-readonly="true"></div>
    <!---->
  </div>
  <!---->
  <div class="dropdown-menu" style="display: none;" name="zoom-in-top-transition" id="dropdown-menu-f26zq37i2"><span class="arrow"></span> </div>
</div>
`;
